lib/livebook_cli/server.ex

```diff
@@ -67,7 +62,11 @@ defmodule LivebookCLI.Server do
     case check_endpoint_availability(base_url) do
       :livebook_running ->
         IO.puts("Livebook already running on #{base_url}")
-        open_from_options(base_url, opts, extra_args)
+        if length(exta_args) > 1 do
+          print_error("Too many arguments entered. Please ensure only one argument is used to specify the file path and all other arguments are preceded by the relevant switch")
```

Comment:
Instead of handling it here, you can handle it as a catch-all on `open_from_options`. This way there is no duplication either.

Reply:
got it :)